Make it clear that you need to enable desktop notifications in Gmail (#325)
timche/gmail-desktop: README.md

- [Menu bar mode](#menu-bar-mode-macos) _(macOS)_
- Unread badge in dock _(macOS)_
- Unread icon in tray _(Linux/Windows)_

> - Desktop notifications (you must [enable desktop notifications in Gmail](https://support.google.com/mail/answer/1075549?co=GENIE.Platform%3DDesktop) for this to work)

Reviewer: ```suggestion
- Desktop notifications (["New mail notifications on" must be enabled](https://support.google.com/mail/answer/1075549?co=GENIE.Platform%3DDesktop) in Gmail settings)
```
